.github/workflows/brew.yml

```diff
@@ -0,0 +1,21 @@
+name: brew
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
should we trigger on a release instead? https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
Yeah, totally, let me update.